phases/13-tools-and-protocols/09-mcp-transports/code/main.py

```
@@ -0,0 +1,229 @@
+        self.send_response(200)
+        self.send_header("Content-Type", "text/event-stream")
+        self.send_header("Mcp-Session-Id", sid)
+        self.send_header("Cache-Control", "no-cache")
+        self.end_headers()
+        for i in range(3):
+            payload = json.dumps({"jsonrpc": "2.0", "method": "notifications/progress",
+                                  "params": {"progressToken": "p1", "progress": i, "total": 3}})
+            self.wfile.write(f"id: {i}\nevent: message\ndata: {payload}\n\n".encode())
+            try:
+                self.wfile.flush()
+            except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)